core/templates/dev/head/components/question-directives/question-player/question-player.directive.ts

```diff
@@ -218,7 +210,8 @@ angular.module('oppia').directive('questionPlayer', [
             var worstSkillId = '';
             Object.keys(ctrl.scorePerSkillMapping).forEach(function(skillId) {
               var skillScoreData = ctrl.scorePerSkillMapping[skillId];
-              var scorePercentage = skillScoreData.score / skillScoreData.total;
+              var scorePercentage =
+              skillScoreData.score / skillScoreData.total;
```

Comment:
Indent by 2 and put parens around it (with '(' on the end of the previous line).

That said, I don't understand why this was changed anyway. It seems to have fit fine on one line before? (Ditto in various places below.)

Reply:
My bad, fixed.